examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/WebDemoWithEventHub.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="4.0.0-preview" />
-    <PackageReference Include="Azure.Identity" Version="1.1.1" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.1" />
-    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="5.0.0" />
+    <PackageReference Include="Azure.Identity" Version="1.6.0" />
```

Comment:
There is an open issue with Azure.Identity v1.5.0 and higher: https://github.com/Azure/azure-sdk-for-net/issues/24767 
I dont know if we want to stick to 1.4.1 for this example or upgrade.

Reply:
Rolled back to 1.4.1. Thanks.